.github/release-drafter-config.yml

```diff
@@ -0,0 +1,55 @@
+name-template: '$NEXT_MINOR_VERSION'
+tag-template: 'v$NEXT_MINOR_VERSION'
+filter-by-commitish: true
+commitish: master
+autolabeler:
+  - label: 'maintenance'
+    files:
+      - '*.md'
+      - '.github/*'
+  - label: 'bug'
+    branch:
+      - '/bug-.+'
+  - label: 'maintenance'
+    branch:
+      - '/maintenance-.+'
+  - label: 'feature'
+    branch:
+      - '/feature-.+'
+categories:
+  - title: '🔥 Breaking Changes'
+    labels:
+      - 'breakingchange'
+  - title: '🧪 Experimental Features'
+    labels:
+      - 'experimental'
+  - title: '🚀 New Features'
+    labels:
+      - 'feature'
+      - 'enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'fix'
```

Comment:
curious why you have all the different variant labels here, in the way I've used labels on github the repo owner makes the set of labels and then most users are just selecting from them in which case the spelling would always be the same.

Reply:
It's mostly history. I'd *love* folks to only use one label. I've watched labels come and go historically, plus be different across repositories. I'm happy to enforce a specifically named label, and in other projects - but this allows for flexibility until we get there.  Something similar happens with branches - human behaviour.